src/System.Net.Http/tests/StressTests/HttpStress/ClientOperations.cs

```diff
@@ -75,22 +79,33 @@ public async Task<HttpResponseMessage> SendAsync(HttpRequestMessage request, Htt
             }
         }
 
-        public string GetRandomString(int maxLength)
+        /// Gets a random ASCII string within specified length range
+        public string GetRandomString(int minLength, int maxLength, bool alphaNumericOnly = true)
         {
-            int length = _random.Next(0, maxLength);
+            int length = _random.Next(minLength, maxLength);
```

Comment:
Looking at some of the places in code where `GetRandomString` is called, it looks like they expect `maxLength` to be inclusive.

Reply:
the upper bound values in `System.Random` apis are generally exclusive, so probably makes sense to preserve that convention.